test/e2e/framework.go

```diff
@@ -975,6 +975,21 @@ func (data *TestData) createBusyboxPodOnNode(name string, ns string, nodeName st
 	return data.createPodOnNode(name, ns, nodeName, busyboxImage, []string{"sleep", strconv.Itoa(sleepDuration)}, nil, nil, nil, hostNetwork, nil)
 }
 
+// getClickHouseOperator retrieves the name of the ClickHouse Operator Pod (clickhouse-operator-*) running on a specific Node.
+func (data *TestData) getClickHouseOperator() (*corev1.Pod, error) {
+	listOptions := metav1.ListOptions{
+		LabelSelector: "app=clickhouse-operator",
+	}
+	pods, err := data.clientset.CoreV1().Pods(kubeNamespace).List(context.TODO(), listOptions)
+	if err != nil {
+		return nil, fmt.Errorf("failed to list Flow Aggregator Pod: %v", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to list ClickHouse Operatorr Pod: %v", err)
```

Reply:
Thanks Yuntang! I finally removed the changes in e2e tests and added a discussion below.